excavator/utils.py

```diff
@@ -1,12 +1,23 @@
 import os
 
 
-def env_string(name, default='', required=False):
+class Empty(object):
+    pass
+
+
+empty = Empty()
```

Comment:
I actually went the inverse way and changed the class to be `empty` and just used the class as the singleton value.

Chose this approach based on what DRF does.  https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/fields.py#L29